fastlane/appstoreres/jetpack_metadata/source/subtitle.txt

```diff
@@ -0,0 +1 @@
+Make your WordPress site faster and safer.
```

Comment:
I came up with this because it's required by release toolkit, and it's anyway a good idea to have it.

I'm sure there's something better we can use, but maybe we'll save it for the next release? This is please is better than having no subtitle, which is the current state.

<img width="657" alt="Screen Shot 2021-06-18 at 4 39 48 pm" src="https://user-images.githubusercontent.com/1218433/122517833-d63b3700-d053-11eb-90a8-2ba8c22ff5ad.png">

@leandroalonso please advise 😄 🙇

Reply:
What's the char limitation of this field? I think we can use `Get powerful security and performance tools in your pocket.` which is the JPAndroid subtitle. :)

Reply:
@leandroalonso Apparently 30, according to the number at the bottom right of this textfield. Which means your suggestion doesn't fit 😢 

![image](https://user-images.githubusercontent.com/216089/122807977-8915ca00-d2cc-11eb-8073-988267489d89.png)